sgpt/cache.py

```diff
@@ -33,11 +33,14 @@ def wrapper(*args, **kwargs):
             cache_key = md5(json.dumps((args[1:], kwargs)).encode('utf-8')).hexdigest()
             cache_file = self.cache_path / cache_key
             if cache_file.exists():
```

Comment:
Above `caching = False` case should be fixed as well, since `_request` now do yeild.